fe1-web/src/features/popcha/README.md

```diff
@@ -0,0 +1,6 @@
+# PoPCha
+
+Provides a scanner that allows the user to scan a QR code to authenticate to a 3rd party website that it belongs to a specific LAO. (Maybe later, will display all tokens the user has, to be able to revoked them).
```

Comment:
```suggestion
Provides a scanner that allows the user to scan a QR code to authenticate to a 3rd party website that it belongs to a specific LAO. (Maybe later, will display all tokens the user has, to be able to revoke them).
```
It could be nice to add a date and a GitHub username when adding comments that use "later" or "for now" to know when you wrote it. You may add something like `(dayan9265, 05-2023)`.